<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor de actualización</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0" colspan="3">
        <widget class="QPushButton" name="ctrlButtonEraseProgVerify">
         <property name="text">
          <string>Borrar-Programar-Verificar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ctrlButtonBootloaderVer">
         <property name="text">
          <string>Version Bootloader</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ctrlButtonLoadHex">
         <property name="text">
          <string>Abrir Archivo Hex</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="ctrlButtonVerify">
         <property name="text">
          <string>Verificar</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="ctrlButtonProgram">
         <property name="text">
          <string>Programar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="ctrlButtonRunApplication">
         <property name="text">
          <string>Iniciar Aplicación</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="ctrlButtonErase">
         <property name="text">
          <string>Borrar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="lblEstado">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Estado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBuscar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
